api/src/auth.js

```diff
@@ -237,15 +241,17 @@ class Auth {
   }
 
   async signinToken(req, res) {
-    const { error, value } = Joi.object({ token: Joi.string().required() }).validate({ token: req.cookies.jwt });
+    const { error, value } = Joi.object({ token_ref: Joi.string(), token_young: Joi.string() }).validate({ token_ref: req.cookies.jwt_ref, token_young: req.cookies.jwt_young });
```

Comment:
```suggestion
    const { error, value } = Joi.object({
      token_ref: Joi.string().allow('', null),
      token_young: Joi.string().allow('', null)
    }).or('token_ref', 'token_young').validate({ token_ref: req.cookies.jwt_ref, token_young: req.cookies.jwt_young });
```

Pour harder une rétrocompatibilité on pourrait s'assurer qu'un des deux est bien présent au moins.

EDIT: ah non on s'en fout tu le teste plus bas 😶